src/ascii/port/callbacks.rs

```diff
@@ -0,0 +1,121 @@
+//! Callbacks for ports.
+#[cfg(doc)]
+use super::Port;
+use super::{Alert, Direction};
+
+/// A callback that is called after a packet is either transmitted or received.
+///
+/// See [`Port::set_packet_handler`] for more details.
+pub type PacketCallback<'a> = Box<dyn FnMut(&[u8], Direction) + 'a>;
+
+/// The same as [`PacketCallback`] but also implements `Send`.
+///
+/// See [`Port::set_packet_handler`] for more details.
+pub type PacketCallbackSend<'a> = Box<dyn FnMut(&[u8], Direction) + Send + 'a>;
```

Comment:
Naming options:
* Suffix (current): `PacketHandlerSend`
* Prefix: `SendPacketHandler`
  * Sounds like it handles sending packets, which isn't true.
* Different adjective: `SendablePacketHandler`
  * Maybe?
* ...?

Reply:
The callback type is `SendCallbacks`, consistency with that may be good.

Reply:
See https://github.com/stphnt/zproto/pull/158#discussion_r1631212834